spec/helpers/builders.coffee

```diff
@@ -1,3 +1,16 @@
+$ = require 'jquery'
+_ = require 'underscore'
+inflection = require 'inflection'
+
+StorageManager = require '../../src/storage-manager'
+
+Post = require './models/post'
+Project = require './models/project'
+Task = require './models/task'
+TimeEntry = require './models/time-entry'
+User = require './models/user'
+
+
 window.spec ?= {}
```

Comment:
Do we still need this now that everything is a module? Could it possibly be a module itself so we can prevent references to global objects?